equipment_handler/source/spi_io.c

```diff
@@ -31,29 +29,30 @@ void SPI_Release(void) {
 
 inline void SPI_CS_Low(uint8_t bVolNum) {
     if (bVolNum == 0) {
-#ifdef IS_ATHENA
-        gioSetBit(gioPORTA, 3, 0); // CS LOW
-        gioSetBit(hetPORT2, 6, 1); // CS HIGH
-#else
-        gioSetBit(hetPORT1, 12, 0); // CS LOW
-        gioSetBit(hetPORT1, 14, 1); // CS HIGH
+    #ifdef IS_ATHENA
+        //gioSetBit(gioPORTA, 3, 0); //CS LOW
```

Comment:
Why is this commented out?

Reply:
This is what we did when we were trying to get the SD card to work. @joshdellaz should we keep this?

Reply:
This is supposed to set the CS for both pins because we have two SD cards

Reply:
I will uncomment this and then make sure the file system is still working today.